timeseries/tests/unittests/models/test_sktime.py

```diff
@@ -64,9 +64,6 @@ def test_when_sktime_converts_from_dataframe_then_data_not_duplicated_and_index_
         )
     )
 
-    # data is not copied
-    assert df.values.base is sktime_df.values.base
-
```

Comment:
Creating a `TimeSeriesDataFrame` now results in copying the data because the sort is always performed.

We could try to only do sorting if the timestamps are not already sorted, but I think that it would be more cumbersome & less efficient (since we need to check if the timestamps are sorted for each individual time series).